### PR TITLE
add chakra

### PR DESCRIPTION
**Pull request by jmzwar**
Add chakra and install theme from v3. Note this is to merge into a branch for refresh work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,11 +62,11 @@
     "lodash": "^4.17.20",
     "next": "^12.2.2",
     "next-compose-plugins": "^2.2.1",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply package.json:65]: Wow I'm surprised we didn't run into any type errors.

File: content/App.tsx [hunk 2]
```diff
@@ -115,17 +118,32 @@ const App: FC<AppProps> = (props) => {
...
-                <InnerApp {...props} />
-              </MediaContextProvider>
-            </WithAppContainers>
-          </QueryClientProvider>
-        </RecoilRoot>
-      </ThemeProvider>
+      <ChakraProvider
+        theme={extendTheme({
+          ...chakraTheme,
+          // TODO: Global V2 Styles, need replacing
```
Comment (bachstatter) [inline content/App.tsx:124]: Should we do this todo before merging?

Comment (jmzwar) [reply content/App.tsx:124]: I think it makes sense to remove this once we have removed the themeprovider that comes with styled-components. Chakra is just overwriting some styles on the body currently.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmzwar) [reply package.json:65]: No issues with building. The next upgrade helped us here.

**Pull request merged**